lib/browser/rpc-server.js

```diff
@@ -146,6 +146,27 @@ const throwRPCError = function (message) {
   throw error
 }
 
+const rendererMissingErrorMessage = (meta, args, callIntoRenderer) => {
+  let message = `Attempting to call a function in a renderer window that has been closed or released.` +
+    `\nFunction provided here: ${meta.location}`
+
+  if (args.length === 0) return message
+  if (!args[0].sender || !args[0].sender._events) return message
+
+  const eventsAttached = args[0].sender._events
```

Comment:
We could, but we'll get all the events. We need to compare the event handler with `callIntoRenderer` to know if it's a remote listener, so we'll still have to iterate.